soh/src/overlays/actors/ovl_En_Box/z_en_box.c

```diff
@@ -645,7 +647,8 @@ void EnBox_UpdateSizeAndTexture(EnBox* this, PlayState* play) {
         }
     }
 
-    if (play->sceneNum != SCENE_TAKARAYA && (cvar == 1 || cvar == 3) && ((agonyCVar > 0 && stoneCheck) | agonyCVar == 0)) {
+    // Change size
+    if (!isVanilla && (cstmc == 1 || cstmc == 3)) {
```

Comment:
not in-scope for this PR, but figure this is probably something we shouldn't use magic numbers for https://github.com/HarbourMasters/Shipwright/issues/2656